ci-operator/step-registry/code-ready/snc/microshift/test/code-ready-snc-microshift-test-commands.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+set -euo pipefail
```

Comment:
Quite sad about 
```
$ diff -u ci-operator/step-registry/code-ready/snc/e2e/test/code-ready-snc-e2e-test-commands.sh  ci-operator/step-registry/code-ready/snc/microshift/test/code-ready-snc-microshift-test-commands.sh
--- ci-operator/step-registry/code-ready/snc/e2e/test/code-ready-snc-e2e-test-commands.sh	2021-11-17 17:08:57.490395679 +0100
+++ ci-operator/step-registry/code-ready/snc/microshift/test/code-ready-snc-microshift-test-commands.sh	2023-04-18 10:58:46.318795740 +0200
@@ -42,7 +42,7 @@
   pushd snc
   set -e
   export OPENSHIFT_PULL_SECRET_PATH="${HOME}"/pull-secret
-  ./ci.sh
+  ./ci_microshift.sh
   popd
 }
 
$ wc -l ci-operator/step-registry/code-ready/snc/microshift/test/code-ready-snc-microshift-test-commands.sh 
80 ci-operator/step-registry/code-ready/snc/microshift/test/code-ready-snc-microshift-test-commands.sh
```

Other jobs refer to scripts in a `${SHARED_DIR}` directory, I have no idea if this could be used to reuse common functionalities between our jobs.